ui/app/index.html

```diff
@@ -46,8 +46,11 @@
             <li ng-class="{ active: $state.includes('home.streams') }">
                <a ui-sref="home.streams.tabs.definitions">Streams</a>
             </li>
+            <li ng-class="{ active: $state.includes('home.tasks') }">
+              <a ui-sref="home.tasks.tabs.definitions">Tasks</a>
+            </li>
             <li ng-class="{ active: $state.includes('home.jobs') }">
```

Comment:
What is the reason behind keeping the `jobs` tab with the **executions** page in it? I thought we are going to have the detailed `executions` page under the `tasks -> executions` anyway and this tab can be removed?